dev-client/src/screens/SitesScreen/components/EmptySiteMessage.tsx

```diff
@@ -14,35 +14,68 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program. If not, see https://www.gnu.org/licenses/.
  */
-import {Trans} from 'react-i18next';
 
-import {BulletList} from 'terraso-mobile-client/components/BulletList';
-import {LocationIcon} from 'terraso-mobile-client/components/icons/LocationIcon';
+import {useTranslation} from 'react-i18next';
+import {StyleSheet} from 'react-native';
+
+import MaterialCommunityIcons from '@expo/vector-icons/MaterialCommunityIcons';
+
+import {TranslatedParagraph} from 'terraso-mobile-client/components/content/typography/TranslatedParagraph';
+import {Icon} from 'terraso-mobile-client/components/icons/Icon';
 import {Text, View} from 'terraso-mobile-client/components/NativeBaseAdapters';
+import {convertColorProp} from 'terraso-mobile-client/components/util/nativeBaseAdapters';
+import {useIsOffline} from 'terraso-mobile-client/hooks/connectivityHooks';
+import {OfflineMessageBox} from 'terraso-mobile-client/screens/LocationScreens/components/soilId/messageBoxes/OfflineMessageBox';
 
 export const EmptySiteMessage = () => {
+  const {t} = useTranslation();
+  const isOffline = useIsOffline();
+
   return (
     <View px="17px">
-      <Text variant="body1">
-        <Trans
-          i18nKey="site.empty.info"
-          components={{
-            bold: <Text bold />,
-          }}
-        />
+      {isOffline ? <OfflineMessageBox message={t('site.offline')} /> : <></>}
+
+      <Text bold mt={isOffline ? 5 : undefined}>
+        <TranslatedParagraph i18nKey="site.empty.info" />
       </Text>
 
-      <BulletList
-        data={[1, 2, 3]}
-        renderItem={i => (
-          <Text variant="body1" color="text.primary">
-            <Trans
-              i18nKey={`site.empty.bullet_${i}`}
-              components={{icon: <LocationIcon />}}
-            />
-          </Text>
-        )}
-      />
+      <View style={styles.view}>
+        <MaterialCommunityIcons
+          name="gesture-tap"
+          size={24}
+          color={convertColorProp('text.icon')}
+          style={styles.communityIcon}
+        />
+        <TranslatedParagraph i18nKey="site.empty.map" />
+      </View>
+
+      <View style={styles.view}>
+        <Icon name="my-location" color="text.icon" size={24} pr={5} />
+        <TranslatedParagraph i18nKey="site.empty.icon" />
+      </View>
+
+      <View style={styles.enter}>
+        <Icon name="search" color="text.icon" size={24} pr={5} marginTop={1} />
+        <TranslatedParagraph i18nKey="site.empty.coords" />
```

Comment:
Issue: Hmm, looks like the text wrapping point is further to the right than we want ("the" should probably wrap):
<img width="403" alt="Screenshot 2024-12-04 at 12 22 34 PM" src="https://github.com/user-attachments/assets/bc4020cd-8650-45e5-99e7-cf167da3b058">